public/coffee/ide/editor/directives/aceEditor/labels/LabelsManager.coffee

```diff
@@ -28,37 +26,26 @@ define [
 					((_.any(change.lines, (line) -> line.match(/\\label\{[^\}\n\\]{0,80}\}/))) or
 					 (commandFragment?.length > 2 and commandFragment.slice(0,7) == '\\label{'))
 				)
```

Comment:
It took me a while to parse and figure out what this was doing. Could change it to something like:

```
return if change.action not in ['remove', 'insert']
changeUpdatedFullLabel = (_.any(change.lines, (line) -> line.match(/\\label\{[^\}\n\\]{0,80}\}/)))
isTypingInLabel =  (commandFragment?.length > 2 and commandFragment.slice(0,7) == '\\label{')
if (changeUpdatedFullLabel or isTypingInLabel)
    ...
```